src/main/scala/QuartzSchedules.scala

```diff
@@ -85,6 +89,10 @@ sealed trait QuartzSchedule {
   //than one calendar anyways.
   def calendar: Option[String]
 
+  def timezone: TimeZone
+
+  def triggerDescription: Option[String]
+
```

Comment:
What's the value in adding these fields here?

Reply:
To make the code QuartzSchedules implementation agnostic.